zellij-server/src/tab/mod.rs

```diff
@@ -235,6 +235,9 @@ pub trait Pane {
     fn push_right(&mut self, count: usize);
     fn pull_left(&mut self, count: usize);
     fn pull_up(&mut self, count: usize);
+    fn clear_screen(&mut self) {
+        todo!();
+    }
```

Comment:
Why not forgo the default implementation here? If you're concerned about plugins (our only other type of `Pane`) then I think we can leave it and remove the `todo!()` - I don't think this action is relevant for plugins...

Reply:
Not really when I was doing this specifically I was trying to understand the whole architecture of zellij, I think I can leave all the default stuff here without conditional you mentioned